Modules/Scripted/DICOMLib/Resources/DICOM/dcmtk/storescu.cfg

```diff
@@ -0,0 +1,62 @@
+#
+#  Copyright (C) 2003-2021, 3DSlicer
+#
+#  Author:  Rob Oakes
+#
+#  Purpose: Storescu configuration optimized for DICOM-SR and DICOM-SEG files
+#
+
+# ============================================================================
+[[TransferSyntaxes]]
+# ============================================================================
+
+[Uncompressed]
+TransferSyntax1 = LittleEndianExplicit
+TransferSyntax2 = BigEndianExplicit
+TransferSyntax3 = LittleEndianImplicit
+
+[JPEGBaseline]
+TransferSyntax1 = JPEGBaseline
+
+[JPEGLossless]
+TransferSyntax1 = JPEGLossless:Non-hierarchical-1stOrderPrediction
+
+[RLE]
+TransferSyntax1 = RLELossless
+
+[MPEG2]
+TransferSyntax1 = MPEG2MainProfile@MainLevel
+#
+# commented out the following line since we do not support transcoding of MPEG2
+#
+#TransferSyntax2 = MPEG2MainProfile@HighLevel
+
+[MPEG4]
+TransferSyntax1 = MPEG4HighProfile/Level4.1
+#
+# commented out the following lines since we do not support transcoding of MPEG4
+#
+#TransferSyntax2 = MPEG4BDcompatibleHighProfile/Level4.1
+#TransferSyntax3 = MPEG4HighProfile/Level4.2For2DVideo
+#TransferSyntax4 = MPEG4HighProfile/Level4.2For3DVideo
+#TransferSyntax5 = MPEG4StereoHighProfile/Level4.2
+
+# ============================================================================
+[[PresentationContexts]]
+# ============================================================================
+
+[StorageCompressedAndUncompressed]
+#
+# For DICOM-SR and DICOM-SEg SOP classes, only uncompressed transmission is supported.
```

Comment:
I don't think SEG is a DICOM term (SR is). In the DICOM standard it is referred to as `Segmentation`.

Reply:
I think you're right. The standard refers to `Segmentation` templates for SR documents, and DICOM-SEG is technically a special type of Structured Report. Because of their role within many image processing applications, though, many programs refer to them separately from SR, though.

You can find references, for example, to DICOM-SEG in the documentation from libraries such as [PyDicom Seg](https://github.com/razorx89/pydicom-seg), [highdicom](https://github.com/razorx89/pydicom-seg), and OHIF.

Reply:
I'm just curious, is there a formal specification of a SR template called DICOM-SEG?

Reply:
No, I don't think there is. You are right, the template seems to be "Segmentation."

There are some places where `SEG` is used distinctively to differentiate segmentations from more general structured reports or annotations. [`SEG` is also used to specify the modality of the file](http://dicom.nema.org/medical/Dicom/2018d/output/chtml/part03/sect_C.8.20.html) in comparison to [`SR` for general structured report instances or annotations](https://dicom.innolitics.com/ciods/cr-image/general-series/00080060) or [`REG` for spatial registration files](http://dicom.nema.org/dicom/2013/output/chtml/part03/sect_C.20.html).

The practice of calling the resources DICOM-SEG, DICOM-REG, etc seems to pretty common, at least amongst the software development communities where I am active. (Maybe taken from the modality?) It also appears in semi-formal resources, such as the writings and [presentations of David Clunie](https://cdn.ymaws.com/siim.org/resource/resmgr/mimi18/presentations/18cmimi_ml-clunie.pdf), so I take it as an "informal semi-canonical" thing.